#2066 Test to cover reported issue with reassign MSI source

lbajsarowicz opened the pull request:
### Description
MFTF test case to show the issue with reassign source.

### Relevant Issues
1. magento-engcom/msi#2066

### Relevant PRs
1. magento-engcom/msi#2069

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

Commit:
#2066 Test to cover reported issue with reassign MSI source

ishakhsuvarov commented:
Hi @lbajsarowicz 

Thank you for resubmitting the PR. 

Unfortunately, we are still not ready to merge it in this state, as issue in question is not yet fixed. So, per our previous conversation, let's update it to the skipped state and get it merged.

ishakhsuvarov committed:
Skip test temporarily

 - Added Skip with link to the appropriate ticket
 - Added test case id covering this item

ghost commented:
Hi @lbajsarowicz, thank you for your contribution! </br>Please, complete [Contribution Survey](https://opensource.magento.com/survey/contribution/magento-engcom+msi+2312), it will take less than a minute. </br>Your feedback will help us to improve contribution process.
